src/main/java/bio/terra/pdao/bigquery/BigQueryPdao.java

```diff
@@ -560,10 +579,51 @@ CROSS JOIN UNNEST(S.refColumnName) AS refColumnName
         return refIdArray;
     }
 
+    public Set<String> getRowIds(Dataset dataset, String tableName, String projectId, BigQueryProject bigQueryProject) {
```

Comment:
If I understand correctly, this is test code. Not sure why it is in the main code tree.
I think it should be in the test code.